src/ndarray/ndarray.cc

```diff
@@ -312,6 +312,46 @@ NDArray NDArray::data_ndarray() const {
   return ret;
 }
 
+struct NDArrayDLManager {
+    NDArray handle;  // ref NDArray
+    DLManagedTensor tensor;
+    TShape strides;  // store variable strides
```

Comment:
DLPack allow strides field to be nullptr,  for mxnet do do not need to store strides, as we only support compact tensor for now.

Reply:
During conversion to mxnet tensor, check the strides field and see if it is compact

Reply:
I will fix it.
It seems that PyTorch.utils.from_dlpack doesn't allow strides nullptr.